controllers/apidefinition_controller.go

```diff
@@ -95,6 +95,9 @@ func (r *ApiDefinitionReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		if desired.Spec.APIID == "" {
 			desired.Spec.APIID = encodeNS(req.NamespacedName.String())
 		}
+		if desired.Spec.OrgID == "" {
+			desired.Spec.OrgID = r.Env.Org
+		}
```

Comment:
Sanity check - Does this not mutate the apidefinition spec - causing a reconciliation?

Reply:
e.g. we don't want this to happen again: https://github.com/TykTechnologies/tyk-operator/issues/250